ang/civiawards/shared/configs/case-type-buttons.config.js

```diff
@@ -0,0 +1,51 @@
+(function (angular, _, url) {
+  var module = angular.module('civiawards');
+  var AWARDS_CATEGORY_NAME = 'awards';
+  var AWARD_CONFIG_URL = 'civicrm/a/#/awards/';
+
+  module.config(function (CaseTypeButtonsProvider) {
+    var awardCategory = getAwardCategory();
+    var awardCaseTypes = getCaseTypesForCategory(awardCategory.value);
+
+    addConfigurationButtonsToCaseTypes(awardCaseTypes);
+
+    /**
+     * Adds configuration buttons to the given case types. The buttons url point
+     * towards the awards configuration form.
+     *
+     * @param {object[]} caseTypes a list of case types objects.
+     */
+    function addConfigurationButtonsToCaseTypes (caseTypes) {
+      _.forEach(caseTypes, function (caseType) {
+        var caseTypeConfigUrl = url(AWARD_CONFIG_URL + caseType.id);
```

Comment:
Also right now, the buttons would be visible for Civicase/Prospect Dashboard too right? we need to implement services for each button.

Reply:
As discussed in Slack the icons won't be visible in other extensions because each one of them filter the case types they show on the dashboard. ie: cases only show cases case type, award dashboard only shows award case types, etc.

Since we are only adding the icons to the award case types we don't need any further filtering.